### PR TITLE
fix my last PR (#2)

### PR DESCRIPTION
**Pull request by guillemap**
To be closer to sublime's Mariana theme for:

- Python
- TypeScript & AssemblyScript
- json
- Graphql
- Solidity
- csv

Minor UI improvements (text selection + debug bar)

### CODE REVIEW

File: themes/mariana-reference.json [hunk 1]
```diff
@@ -246,10 +273,35 @@
...
+				"meta.fstring.python constant.character.format.placeholder"
+			],
+			"settings": {
+				"foreground": "LIGHT_1"
+			}
+		},
+		{
+			"name": "Class",
+			"scope": [
+				"entity.name.type.class"
```
Comment (mightbesimon) [inline themes/mariana-reference.json:294]: this duplicates `entity.name` which is already orange

**Comment by mightbesimon** [comment]
Hi @guillemap, thank you so much for the contribution. To make reviewing easier and faster for me, could you please provide some screenshots of the syntax highlight before and after? I'll try get this merged this week and let you know what changes (if any) are necessary.
Thanks again!

**Comment by mightbesimon** [comment]
I really like the update on decorators, great catch! I've not realised until now

File: themes/mariana-reference.json [hunk 2]
```diff
@@ -153,7 +154,7 @@
 		"statusBar.foreground": "LIGHT_1",
 		"statusBar.background": "DARK_0",
 		"statusBar.debuggingBackground": "ORANGE",
-		"statusBar.debuggingForeground": "DARK_0",
+		"statusBar.debuggingForeground": "DARK_1",
```
Comment (mightbesimon) [inline themes/mariana-reference.json:157]: I could be convinced of changing to `DARK_1`. I chose `DARK_0` for maximum foreground background contrast.

File: themes/mariana-reference.json (hunk 1 above)
Comment (guillemap) [reply themes/mariana-reference.json:294]: Not really, this overrides `entity.name.type` which is blue & italic, but in Python classes it should be orange & upright

File: themes/mariana-reference.json (hunk 2 above)
Comment (guillemap) [reply themes/mariana-reference.json:157]: Agreed!

**Comment by guillemap** [comment]
Some screenshots. On left Sublime, on right VS (first with the current version, then with the PR implemented)

0a
0z
1a
1z
2a
2z
3a
3z
4a
4z
5a
5z
6a
6z

Plus a UI improvement when selecting highlighted words
select

**Comment by guillemap** [comment]
Rust before
Captura de pantalla 2023-03-02 a les 2 05 11

Rust after
Captura de pantalla 2023-03-02 a les 2 42 16

What's not identical is due to same identifiers used in different contexts (String in line 9/41 and params in line 25). Left what looked more appealing when actually coding.

**Comment by mightbesimon** [comment]
could you do a separate PR for the rust changes? I'm more open to approve the rust changes than all changes bundled in this one PR, specifically around the `editor.<stuff>` ones and python ones.

**Comment by mightbesimon** [comment]
similarly also separate for graphql, solidity, and rainbow csv

**Comment by guillemap** [comment]
Separated into many PR's as requested:
https://github.com/mightbesimon/vscode-mariana/pull/8 https://github.com/mightbesimon/vscode-mariana/pull/9 https://github.com/mightbesimon/vscode-mariana/pull/10 https://github.com/mightbesimon/vscode-mariana/pull/11 https://github.com/mightbesimon/vscode-mariana/pull/12 https://github.com/mightbesimon/vscode-mariana/pull/13 https://github.com/mightbesimon/vscode-mariana/pull/14 https://github.com/mightbesimon/vscode-mariana/pull/15

**Pull request closed without merging**